pkg/cmdutil/features.go

```diff
@@ -0,0 +1,20 @@
+package cmdutil
+
+import (
+	"os"
+)
+
+func AdvancedIssueSearchEnabled(f *Factory) bool {
+	advancedSearch := os.Getenv("GH_ADVANCED_ISSUE_SEARCH")
+
+	if advancedSearch == "" {
+		config, err := f.Config()
+		if err != nil {
+			advancedSearch = "disabled"
+		} else {
+			advancedSearch = config.AdvancedIssueSearch("").Value
```

Comment:
leaving a comment that we actually want to check the hostname as this could be set per host